bullet_train-themes-light/app/assets/stylesheets/light/fields/super_select.css

```diff
@@ -109,7 +109,7 @@
 
   .select2-search__field {
```

Comment:
Not sure which selector needs to be targeted yet, but there's a blue that's defined in select2.css that's not being overridden anywhere.

<img width="1480" height="642" alt="CleanShot 2025-10-29 at 18 23 02@2x" src="https://github.com/user-attachments/assets/2a656628-e5d9-42c5-b4fe-0295041e7865" />